crates/frui_widgets/src/container.rs

```diff
@@ -64,10 +68,52 @@ impl<W: Widget> RenderWidget for Container<W> {
     fn paint(&self, ctx: &mut PaintCtx<Self>, canvas: &mut Canvas, offset: &Offset) {
         if let Some(color) = &self.color {
             let brush = &canvas.solid_brush(color.clone());
-
-            RenderContext::fill(canvas, Rect::from_origin_size(offset, ctx.size()), brush);
+            canvas.fill(DruidRect::from_origin_size(offset, ctx.size()), brush);
         }
 
         ctx.child(0).paint(canvas, offset)
     }
 }
+
+#[derive(RenderWidget, Builder)]
+pub struct DecoratedBox<W: Widget, D: Decoration> {
+    pub child: W,
+    pub decoration: D,
+    pub position: DecorationPosition,
+}
```

Comment:
Could you move `DecoratedBox` to `frui_widgets/src/boxes.rs`?

Reply:
I will move these to boxes.rs